Magi/Brunhilde of the Black Forest/Brunhilde, the Corpse Witch.md

```diff
@@ -33,21 +30,20 @@ Brunhilde is a witch living in a cavern, deep inside the Black Forest. She somet
 **Reputations:** The Bone Witch 1 (among the peasants)
 
 **Combat:**  
-*Dodge*: Init -1, Attack +0, Defense -1, Damage +0                                                                                                    
+*Dodge*: Init -1, Attack +0, Defense -1, Damage +0                                                                         
```

Comment:
There seems to be some trailing whitespace here ?

Reply:
Yes, I'll remove them.